### PR TITLE
feat(installer): add etcdbackuper

### PR DESCRIPTION
**Pull request by Huimintai**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: hack/etcdbackuper/values.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+conf:
+  namespace: etcd-ops
+  serviceAccount: etcd-operator
+  clusterRole: etcd-operator-clusterrole
+  clusterRoleBinding: etcd-operator-bind
+  secretName: etcd-v3-secret
+  storageClsName: csi-cephfs-sc 
+  storageRequest: 24Gi
+  image: ccr.ccs.tencentyun.com/lutzowguo/etcd-operator:v0.0.1
+  imagePullPolicy: Always
```
Comment (leoryu) [inline hack/etcdbackuper/values.yaml:11]: we don't need Always

File: hack/etcdbackuper/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+`tdcc-platform-gc` is responsible for gc clustercredential once the cluster is deleted.
+
+## Prerequisites
+
+- Kubernetes 1.18+
+- Helm 3.1.0+
+
+## Installing the Chart
+
```
Comment (leoryu) [inline hack/etcdbackuper/README.md:13]: is this install method works in idc env?

File: hack/etcdbackuper/values.yaml (hunk 1 above)
Comment (Huimintai) [reply hack/etcdbackuper/values.yaml]: Change to IfNotPresent

File: hack/etcdbackuper/README.md (hunk 2 above)
Comment (Huimintai) [reply hack/etcdbackuper/README.md:13]: yes

**Pull request merged**